### PR TITLE
README edits

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request updates our README

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,36 +8,80 @@
...
+- Submit a pull request for review.
 
-We recommend forking our repo, creating a new branch for your content changes, and submitting a pull request. We will help review, test, and merge the content for publishing.
+Our team will help review, test, and merge your contributions for publishing.
 
-## Building Locally
+Sumo Docs is built with [Docusaurus 3](https://docusaurus.io/) and supports React, Rehype, and Remark plugins. We also use [cla-bot](https://colineberhardt.github.io/cla-bot/) to manage our Contributor License Agreement (CLA) process.
 
-Docusaurus requires the following to build on locals:
+Before submitting an issue or pull request, please review the sections below.
```
Comment (JV0812) [inline README.md:26]:
```suggestion
Before submitting an issue or pull request, we recommend you to review the sections below.
```

File: README.md [hunk 2]
```diff
@@ -8,36 +8,80 @@
...
+   cd sumologic-documentation
+   ```
+3. Install dependencies:
+   ```bash
+   yarn install
+   ```
+
+## Apply your changes
+
+Make edits using Markdown syntax. Keep contributions concise, informative, and aligned with our guidelines.
```
Comment (JV0812) [inline README.md:58]:
```suggestion
Make edits using [Markdown syntax](https://help.sumologic.com/docs/dashboards/panels/markdown-syntax/). Keep contributions concise, informative, and aligned with our guidelines.
```

**Review by JV0812 — APPROVED** [review]
LGTM, Great work Kim!

File: README.md [hunk 3]
```diff
@@ -8,36 +8,80 @@
   <a href="https://help.sumologic.com/release-notes-service"><img src="https://img.shields.io/badge/RSS-FFA500?style=for-the-badge&logo=rss&logoColor=white" alt="RSS Follow" width="50"/></a>
 </p>
 
-Share your knowledge with the Sumo Logic community by contributing to our docs! You can contribute by creating an issue or pull request (PR) on our GitHub repository. We welcome all types of contributions; from minor typo fixes to new topics.
+Sumo Docs is the open-source documentation site for Sumo Logic, an all-in-one cloud data analytics platform built to support Security, Operations, and Business Intelligence use cases. Sumo Logic empowers users to monitor, analyze, troubleshoot, and visualize data from their applications and network environments in real time. Its elastic processing capabilities enable seamless log data collection and management from various sources, regardless of type, volume, or location. Learn more at [sumologic.com](https://www.sumologic.com).
```
Comment (jpipkin1) [inline README.md:11]:
```suggestion
Sumo Docs is the open-source documentation site for Sumo Logic, an all-in-one cloud data analytics platform built to support security, operations, and business intelligence use cases. Sumo Logic empowers users to monitor, analyze, troubleshoot, and visualize data from their applications and network environments in real time. Its elastic processing capabilities enable seamless log data collection and management from various sources, regardless of type, volume, or location. Learn more at [sumologic.com](https://www.sumologic.com).
```

File: README.md (hunk 2 above)
Comment (jpipkin1) [reply README.md:58]: I think this would be a better link: https://www.markdownguide.org/tools/docusaurus/

**Review by jpipkin1 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -8,36 +8,80 @@
...
+   cd sumologic-documentation
+   ```
+3. Install dependencies:
+   ```bash
+   yarn install
+   ```
+
+## Apply your changes
+
+Make edits using Markdown syntax. Keep contributions concise, informative, and aligned with our guidelines.
```
Comment (kimsauce) [inline README.md:58]:
```suggestion
Make edits using [Markdown syntax](https://help.sumologic.com/docs/contributing/style-guide/#markdown). Keep contributions concise, informative, and aligned with our guidelines.
```

**Pull request merged**